lib/vdsm/storage/hsm.py

```diff
@@ -2857,6 +2858,25 @@ def measure(self, sdUUID, imgUUID, volUUID, dest_format, backing=True,
 
         return dict(result=result)
 
+    @public
+    def prune_bitmaps(self, sdUUID, imgUUID, volUUID, baseUUID):
+        """
+        Prune stale bitmaps from the base volume.
```

Comment:
Please document that this is an internal that should be used only from virt code.